Makefile

```diff
@@ -77,11 +77,11 @@ clean:
 	cargo clean
 	go clean ./...
 
-# the build-soroban-rpc build target is an optimized build target used by
+# the build-stellar-rpc build target is an optimized build target used by
 # https://github.com/stellar/pipelines/stellar-horizon/Jenkinsfile-soroban-rpc-package-builder
 # as part of the package building.
-build-soroban-rpc: build-libs
-	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o soroban-rpc -trimpath -v ./cmd/soroban-rpc
+build-stellar-rpc: build-libs
```

Comment:
Suggestion: maybe we just rename the repo and output binary to `stellar-rpc` and rename the folders, commands, etc. themselves to just `rpc`? Just in case we need another one of these in the future :p